salt/config/__init__.py

```diff
@@ -944,6 +944,9 @@ def _gather_buffer_space():
     # Always generate minion id in lowercase.
     'minion_id_lowercase': bool,
 
+    # If set, suppress this domain in a generated minion id.
```

Comment:
It's not clear to me from this description that this needs to be a string containing the domain one wishes to suppress. Also, what if somebody wants to suppress multiple domains?

Reply:
I added "single" and an example to the description.
Is it ok when I add more than one line description?

I think one should not suppress multiple domains because the hostname may not be unique in multiple domains.
Stripping multiple domains could lead to name clashes.